src/utils/event.js

```diff
@@ -88,7 +59,7 @@ export const getHumanizeDiffTime = (time) => {
 };
 
 export const getDayEvents = (events) => {
-  events = events.sort((a, b) => a.start - b.start);
+  events = events.slice().sort((a, b) => a.start - b.start);
```

Comment:
Такой слайс применяется, если хочешь заново выделить память под массив.
но мы ж тут все за современный, хипстерский синтаксис.
Давай деструктуризацию юзать
```suggestion
  events = [...events.sort((a, b) => a.start - b.start)];
```

Reply:
sort мутирует исходный массив, slice нужен для получения копии